### PR TITLE
Add getters to Organization for name, token name and symbol

### PR DESCRIPTION
**Pull request by tibetsprague**
Update and fix typescript definitions

Bump npm version

### CODE REVIEW

File: lib/arc.d.ts [hunk 1]
```diff
@@ -507,5 +511,9 @@ declare module "daostack-arc-js" {
       opts: ProposeContributionParams
     ): Promise<TransactionReceiptTruffle>;
     setParams(params: ContributionRewardParams): Promise<string>;
+
```
Comment (dkent600) [inline lib/arc.d.ts]: I'm not sure these events should be part of this PR, better to be part of a broader task aimed at adding better support for events across all of the wrappers.  In any case, I don't see that they've been implemented in the ContributionReward class.

File: tsconfig.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
   "compilerOptions": {
     "module": "commonjs",
     "noImplicitAny": true,
-    "outFile": "./build/tsc.js",
```
Comment (dkent600) [inline tsconfig.json]: What is this for?

File: lib/arc.d.ts [hunk 3]
```diff
@@ -234,6 +234,10 @@ declare module "daostack-arc-js" {
     // vote(proposalId, choice, params);
     static new(options: OrganizationNewConfig): Promise<Organization>;
     static at(avatarAddress: string): Promise<Organization>;
+
+    getName(): string;
+    getTokenName(): string;
```
Comment (dkent600) [inline lib/arc.d.ts]: Can you please add comments describing what these three new methods do? (like *what* name is returned, and so on)?  Thanks.

File: tsconfig.json (hunk 2 above)
Comment (tibetsprague) [reply tsconfig.json]: I couldn't run tsc to test the syntax of the definitions file without removing this

File: lib/arc.d.ts (hunk 1 above)
Comment (tibetsprague) [reply lib/arc.d.ts]: thats true, and this is a good opportunity to do that work, but I needed these definitions to get my old alchemy demo working again. these functions come directly from the truffle contract

File: lib/arc.d.ts (hunk 3 above)
Comment (tibetsprague) [reply lib/arc.d.ts]: sure

File: lib/arc.d.ts (hunk 1 above)
Comment (tibetsprague) [reply lib/arc.d.ts]: are you ok with leaving these in for now while we work out how we want to handle the event functionality in the wrappers? in fact we may decide to just leave as is and pass through to the truffle contract handling for now

Comment (dkent600) [reply lib/arc.d.ts]: sure

File: lib/arc.d.ts (hunk 3 above)
Comment (dkent600) [reply lib/arc.d.ts]: Heh, should be "Organization name", not "DAO"  Sorry ;-)

File: lib/organization.js [hunk 4]
```diff
@@ -274,14 +274,26 @@ export class Organization {
     return this.votingMachine.vote(proposalId, choice, params);
   }
 
+  /**
```
Comment (dkent600) [inline lib/organization.js]: Organization, not DAO here and below too

**Pull request merged**